packages/grafana-ui/src/components/Table/ImageCell.tsx

```diff
@@ -16,7 +16,14 @@ export const ImageCell = (props: TableCellProps) => {
 
   return (
     <div {...cellProps} className={tableStyles.cellContainer}>
-      {!hasLinks && <img src={displayValue.text} className={tableStyles.imageCell} alt="" />}
+      {!hasLinks && (
+        <img
+          style={{ height: tableStyles.cellHeight - DATALINKS_HEIGHT_OFFSET, width: 'auto' }}
+          src={displayValue.text}
+          className={tableStyles.imageCell}
+          alt=""
```

Comment:
Should we remove the superfluous empty `alt` prop? Or better yet, fill it in? 😁

Reply:
Unfortunately there isn't an easy way to fill the alt property as we don't know what the image is 😢. Would be a nice follow-up to add alt and title text to image cell configuration 😄